cloudformation/lambda/geowebcache_seeder/geowebcache_seeder.py

```diff
@@ -80,7 +80,10 @@ def exit_error(msg, err=1):
 
 # AWS Lambda Handler
 def handler(event, context):
-    main()
```

Comment:
@bhasin85 Good news, I found why this is occurring.

Relevant info from: http://click.pocoo.org/5/api/
> This will always terminate the application after a call. If this is not wanted, SystemExit needs to be caught.

The issue was that the main function is raising a SystemExit exception, which Lambda is not expecting.
So rather than the try/finally block, the correct solution as per Click documentation is the following:

```python
try:
    main()
except SystemExit:
    pass
```
This will ignore the SystemExit thrown out of click (when it exits the click entry point), and allow the specific Lambda handler to be the exit point of the program.